src/foam/dao/EasyDAO.js

```diff
@@ -149,9 +149,9 @@ return delegate;
       name: 'innerDAO'
     },
     {
+      /**  Creates pipelinePMDAO's around each decorator to measure their performance */
```

Comment:
documentation: 'Creates pipelinePMDAO's around each decorator to measure their performance'